Tests/SwiftParserTest/Types.swift

```diff
@@ -71,21 +71,21 @@ final class TypeTests: XCTestCase {
                 { $0.parseClosureExpression() },
                 diagnostics: [
                   DiagnosticSpec(locationMarker: "1️⃣", message: "expected identifier in closure capture item"),
-                  DiagnosticSpec(locationMarker: "1️⃣", message: "unexpected text 'class' in closure capture signature"),
+                  DiagnosticSpec(locationMarker: "1️⃣", message: "unexpected code 'class' in closure capture signature"),
```

Comment:
Should we also expand single token keywords? Ie. `class keyword` here rather than `'class'`?

Reply:
Good idea. 👍  I’ll implement it in a follow-up PR.